### DISCUSSION TITLE
Configure asset tagging

### DISCUSSION THREAD

**Discussion by rmoecklideveloper** [atrocore/atrocore#424] [Q&A] [answered]
By default there are 4 tags available tag1, tag2, tag3, tag4. I can't finde the location where you can edit them or add new ones. Is that possible, if yes, could you please share where?

Purpose: To identify different kind of documents like installation guides, tech data sheets etc. and also to add the information of it's language.

Thx

**Comment by marcus-at-localhost** [comment, marked as answer]
You'll find it in the Field Manager: https://demo.atropim.com/#Admin/fieldManager/scope=Asset&field=tags

![image](https://user-images.githubusercontent.com/1337223/183576524-7e974274-2eef-4c7e-9615-446eb0af1642.png)


Same for Products:

https://demo.atropim.com/#Admin/fieldManager/scope=Product&field=tag

![image](https://user-images.githubusercontent.com/1337223/183576029-eaf454c7-c3d3-4195-ad80-906afa66dcda.png)

**Reply by rmoecklideveloper** [reply]
Thx for the fast reply, perfect!
